docs/reference/api/connectivity/networksocket/DNS.md

```diff
@@ -0,0 +1,89 @@
+## DNS
+
+The DNS provides an interface to do DNS host name resolutions.
+
+### DNS class reference
+
+[![View code](https://www.mbed.com/embed/?type=library)](http://os-doc-builder.test.mbed.com/docs/development/mbed-os-api-doxy/class_dns.html)
+
+### Usage
+
+To make a DNS host name resolution: 
+
+1. Instantiate and connect network interface.
+1. Call the `gethostbyname` function to resolve address. 
```

Comment:
Use `()` for functions to make it even more visible. So `gethostbyname()`

Reply:
Corrected.